notebooks/prepare_pa_lov.ipynb

```diff
@@ -1973,6 +1973,23 @@
     "clean_ds[\"value\"].notnull().sum(dim='location_id').plot()"
```

Comment:
<p>Thank you @arun-reddy-a.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/patel-zeel/kdd24/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>